### ISSUE TITLE
little rat is not working on the browser

### ISSUE THREAD

**Issue by pharazone** [dnakov/little-rat#31]
It worked before, but it is not working on the browser now

**Comment by dnakov** [comment]
can you provide some details? screenshots, browser version, OS, etc

**Comment by pharazone** [comment]
![image](https://github.com/user-attachments/assets/f64c949e-a234-4df5-adf0-1b80a32ecc20)
browser : vivaldi,  OS: macOS 15 && windows 11 both

**Comment by Gahwa-Gahwa** [comment]
Same here, not working after updating from Ungoogled Chromium 127 to 129
No longer detects/records web access of extensions

**Comment by dnakov** [comment]
Yeah, unfortunately, it's no longer possible to detect that traffic:
https://chromium-review.googlesource.com/c/chromium/src/+/5636396

**Comment by SullieLore** [comment]
Also works in Edge with this flag enabled: edge://flags/#extensions-on-edge-urls

Why was the app removed from the Chrome store? The store version is still on my machine thankfully.

**Comment by huyz** [comment]
@dnakov can we make it work in Chrome/Brave with a flag? That would be good enough for me

**Comment by dnakov** [comment]
@huyz yeah it works like before if you start the browser with the --extensions-on-chrome-urls flag

**Comment by msenturk** [comment]
it's not working on Chrome with --extensions-on-chrome-urls flag but works on Brave with flag.
